src/vorta/application.py

```diff
@@ -73,7 +76,11 @@ def open_main_window_action(self):
         self.main_window.raise_()
 
     def backup_started_event_response(self):
-        icon = QIcon(get_asset('icons/hdd-o-active.png'))
+        if SettingsModel.get(key='use_light_icon').value:
+            icon = QIcon(get_asset('icons/hdd-o-active-light.png'))
+        else:
+            icon = QIcon(get_asset('icons/hdd-o-active.png'))
+
         self.tray.setIcon(icon)
 
     def backup_finished_event_response(self):
```

Comment:
the icon (which is configured to be "light" for me) turns back to dark due to code below.

Reply:
guess it would be good to have a function like:
```
def get_icon(name, use_style):
    if use_style:
        name = f'{name}-light' if ... else f'{name}-dark'
    return QIcon(get_asset(f'icons/{name}.png'))
```